cns/restserver/const.go

```diff
@@ -38,6 +38,7 @@ const (
 	NetworkContainerVfpProgramComplete     = 35
 	NetworkContainerVfpProgramCheckSkipped = 36
 	NmAgentSupportedApisError              = 37
+	UnsupportedNCVersion                   = 37
```

Comment:
new const?

Reply:
Yep, it's a mistake during resolving merge conflicts. Let me fix it.